static/js/skeleton.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function() {
+
+	setTimeout(function() {
```

Comment:
I had originally planned to use a handlebars conditional to handle rendering the skeleton element vs the real one but turns out that isn't possible. When the page loads, we already have the data e.g. repos. The issue with the delayed rendering appears to be due to how we're looping over that data within the input fields. So, because the data is already there when the page loads, we're simply displaying the skellie stuffs for half a second before moving it and displaying the real element.

Curious on thoughts are the half second wait time. Seems to work for this page but may differ with more data?

Reply:
ahhhh thats inconvenient. So is it the select2 elements that are slow?
They might have an event when they are ready?

Since the repo data gets passed with the template(think theres about a 20 repos max) we should see a pretty consistent load..... we could try force 100+ or so and see how it reacts.

Reply:
Nah it's not just the select element. Repository is janky too and that's just a div but I think it's the looping that's slowing the rendering down.

```
{{#each repos}}
        <div class="hidden default-repos">{{node.full_name}}</div>
      {{/each}}
```

Reply:
Looks like the max we can pass is 100. Just tested it out and still looks good.

Reply:
👍